src/tsdb.c

```diff
@@ -657,12 +658,18 @@ int SeriesCreateRulesFromGlobalConfig(RedisModuleCtx *ctx,
         compactedLabels[labelsCount + 1].value =
             RedisModule_CreateStringPrintf(NULL, "%ld", rule->timeBucket);
 
+        int rules_options = TSGlobalConfig.options;
+        rules_options &= ~SERIES_OPT_DEFAULT_COMPRESSION;
+        rules_options &= SERIES_OPT_UNCOMPRESSED;
+
+        TSGlobalConfig.options &= SERIES_OPT_DEFAULT_COMPRESSION;
```

Comment:
@filipecosta90 im not sure what's the meaning here.
1. we reset the default compression bit
2. this seems like a no-op to me, if its uncompressed it wouldn't do anything, and if it isnt it wouldn't change it.
3. in line 665 we update the global config as part of a key creation, which is probably not the right place.

Reply:
you're right. I missed this after a merge conflict resolution. 
Like we have on master ([link](https://github.com/RedisTimeSeries/RedisTimeSeries/blob/master/src/tsdb.c#L668)) we use uncompressed chunks for rule created series ( keeping the same behavior). 
I've removed the offending lines.